tests/neg/singleton-ops-int.scala

```diff
@@ -70,8 +73,6 @@ object Test {
 
   val t48: ToString[213] = "213"
   val t49: ToString[-1] = "-1"
-  val t50: ToString[0] = "-0" // error
-  val t51: ToString[200] = "100" // error
```

Comment:
> What's wrong with that test?

Nothing was wrong. At first I wanted to remove all `int.ToString` tests because it was deprecated, but then I decided not to remove the tests until it was removed completely. So perhaps I forgot to bring these test back as well. I could change the commit, but I don't think it's a big deal.